### ISSUE TITLE
Transient Embeddings

### ISSUE THREAD

**Issue by samiragarwala** [nerfstudio-project/nerfstudio#2297]
Hi,
Thank you for the great work! I noticed that transient embeddings from NeRF-W seem to have been implemented in semantic NeRF-W with the appropriate loss functions but there is a message noting that they aren't working completely. I was wondering if you could share a bit more about the issues being encountered in getting the transient disentaglement working?
Thank you!

**Comment by kerrj** [comment]
I'm not sure of the history behind this. I think maybe @ethanweber would know?

**Comment by auzyze** [comment]
Hi, is there any update on this question? 
I tried to enable the transient embedding and train on sitcoms-3d dataset. With default settings, the rendered images are very dark during the training process. I modified the beta minimum value but the problem is not solved. By observing the training losses, the rgb loss curve saturated fairly soon, after the uncertainty loss goes down to a certain level. Could you suggest any possible solutions to this issue? Thanks!
